### PR TITLE
improve README order and formatting

### PR DESCRIPTION
**Pull request by bbodenmiller**
- [X] Sign the [electronic OBA ICLA](https://docs.google.com/forms/d/12jV-ByyN186MuPotMvxJtNKtSaGGTnEHm8rXomM2bm4/viewform)

- [X] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,14 +93,13 @@ CAUTION: Every time you re-deploy to Lambda using the `lambduh-maven-plugin`, yo
 manually re-add "Alexa Skills Kit" as the function's _Event Source_.  You do not need to
 do this if you deploy your code through the Lambda UI in AWS Console.
 
-### Authors
+## Authors
```
Comment (barbeau) [inline README.md]: @bbodenmiller Let's just remove the "Authors" section entirely.  We don't have this in any other OBA repo README.

**Comment by barbeau** [comment]
@bbodenmiller Thanks for the cleanup!  One comment inline.

**Comment by bbodenmiller** [comment]
Thanks for the review, updated MR.

**Comment by barbeau** [comment]
@bbodenmiller Thanks, can you please squash to a single commit?

**Comment by barbeau** [comment]
ICLA manually verified :+1:

**Comment by bbodenmiller** [comment]
Moved to https://github.com/OneBusAway/onebusaway-alexa/pull/75.

**Pull request closed without merging**